src/components/Capture/Face.js

```diff
@@ -10,10 +10,9 @@ import withCrossDeviceWhenNoCamera from './withCrossDeviceWhenNoCamera'
 import GenericError from '../GenericError'
 import FallbackButton from '../Button/FallbackButton'
 import CustomFileInput from '../CustomFileInput'
-import { isDesktop, addDeviceRelatedProperties } from '~utils'
+import { isDesktop, addDeviceRelatedProperties, getMobileOSName } from '~utils'
 import { compose } from '~utils/func'
 import { randomId, upperCase } from '~utils/string'
-import { getMobileOSName } from '~utils/detectMobileOS'
```

Comment:
Are we deleting this function in utils/detectMobileOS also?

Reply:
It's been moved to `utils/index.js` to keep all similar utils in the same place so `utils/detectMobileOS` file is now no more.